ome-demoserver.yml

```diff
@@ -293,6 +293,25 @@
         dest: /etc/cron.daily/nightly-pg_dump-omero.sh
         mode: "u=rwx,go="
 
+    - name: Create a figure scripts directory
+      become: yes
+      file:
+        path: /opt/omero/server/OMERO.server/lib/scripts/omero/figure_scripts
+        state: directory
+        mode: 0755
+        recurse: yes
+        owner: "omero-server"
+        group: "omero-server"
+
+    - name: Download the Figure_To_Pdf.py script
+      become: yes
+      get_url:
+        url: https://raw.githubusercontent.com/ome/omero-figure/v3.1.2/omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py
```

Comment:
We'll need to find out if it's safe to mix versions of the figures script with different versions of figure. This figure version "3.1.2" is not hard-coded anywhere else in the playbook, and you may be pulling in a newer version of the script than the accessing OMERO.web client.

@openmicroscopy/devops have also discussed hard-coding the versions of the OMERO.web plugins on each of the systems, so if we go ahead with that strategy, then hard-coding the version here is fine - we'd just need to change it alongside the version of figure. Or have this and the "install figure" statement parse a variable defined further up the playbook with the version defined.

Reply:
@pwalczysko As discussed on one of your other playbooks, you'll need to either add `force: yes` or add a checksum

Reply:
The server doesn't need to edit the location, only be able to read it.

Reply:
@joshmoore : so I should change the line https://github.com/openmicroscopy/prod-playbooks/pull/10/files#diff-896819b5bdf43f197f838c7cc66babf4R311 to 0500 (read and execute by owner), right ?

Reply:
It's not a script in the "linux/execute" sense, so won't need `+x`. And to allow replacement and deletion via CLI, the server might want to be able to write it too, so `0600`?

Reply:
If you're managing this with Ansible the server shouldn't need write access.

Reply:
That's true, provided we don't test it on production then. As it's extra hassle to (remember to) change the perms during a broken production deploy, when I'd expect `omero upload` to work.

Reply:
If production breaks doesn't that imply the playbook is broken and should be fixed?

Reply:
Yes, but coming to that conclusion doesn't help a live out-of-hours deployment. We can go with your suggestion, and hopefully it's tested on a non-prod system then.

Reply:
Ideally this would be refactored so it's testable by travis.

Reply:
If there's a deeper reason you see the figure script as being prone to failures can you add that to a card and we should try and fix it? The whole playbook should be equally trusted for a production server.

Reply:
first test to add is for the script itself so we can test it via travis
the infra-work will help on that front

Reply:
I've just never deployed it like this myself, so don't know what to expect. And I see a mechanism for "escape/fixing" as a sysadmin as being removed by the changing of the permissions.